doc/index.md

```diff
@@ -2,39 +2,61 @@
 
 ## What is Org-roam?
 
-Org-roam is a rudimentary [Roam][roamresearch] replica built around
-the all-powerful [Org-mode][org]. 
+Org-roam is a [Roam][roamresearch] replica built around the
+all-powerful [Org-mode][org].
 
-Like Roam, Org-roam offers a powerful and effortless non-hierarchical
-note-taking approach. With Org-roam, notes flow naturally, making
-note-taking fun and easy. To understand more about Roam, I recommend
-the following links:
+Org-roam is a solution for effortless non-hierarchical note-taking,
+building upon Org-mode. This solution is heavily inspired by [Roam
+Research][roamresearch]. With Org-roam, notes flow naturally, making
+note-taking fun and easy. Org-roam should also work as a plug-and-play
+solution for anyone already using Org-mode for their personal wiki.
 
--   [Building a second brain in
-    Roam](https://reddit.com/r/RoamResearch/comments/eho7de/building_a_second_brain_in_roamand_why_you_might)
--   [Roam: Why I Love It and How I Use
-    It](https://www.nateliason.com/blog/roam)
+To understand more about Roam, a collection of links are available in
+[the appendix](notetaking_workflow.md).
 
-The goal of the project is to implement core features of Roam around
-Org-mode, and eventually introduce features enabled by the Emacs
-ecosystem. 
+Org-roam aims to implement the core features of Roam, leveraging the
```

Comment:
change "aims to implement" to "implements"

Reply:
I'm not sure if we can say it's done implementing the core features of Roam though :sweat_smile: